CHANGELOG.md

```diff
@@ -829,9 +829,9 @@ Story
 **Closed issues:**
 
 - Some broken links on docs-3.0 DHF Tutorial [\#930](https://github.com/marklogic/marklogic-data-hub/issues/930)
-- Data Hub Framework website links still refer to old marklogic-community address [\#899](https://github.com/marklogic/marklogic-data-hub/issues/899)
+- Data Hub website links still refer to old marklogic-community address [\#899](https://github.com/marklogic/marklogic-data-hub/issues/899)
 - Error when harmonising literal in triple [\#878](https://github.com/marklogic/marklogic-data-hub/issues/878)
-- One of the links to the data hub framework website on readme.md is broken [\#864](https://github.com/marklogic/marklogic-data-hub/issues/864)
+- One of the links to the data hub website on readme.md is broken [\#864](https://github.com/marklogic/marklogic-data-hub/issues/864)
```

Comment:
Should be: One of the links to the Data Hub website...
(The "the" is OK here since Data Hub is adjectival.)